collection_modules/btle_beacon/README.md

```diff
@@ -16,6 +16,21 @@ This was built and tuned for the BlueGiga bled112 device.  It has been used with
 - Then ever 3 seconds the beacon is seen within 10m a follow up event is thrown.  
 - When the beacon has not been seen for 20 scans a exit event is thrown.
 
+#### Proximity Gateway Events thrown
+1. `client_in`
+2. `client_out`
+3. `btle_update_nearby`
```

Comment:
i think this name is goofy for an event
how about client_proximity_update or clients_in_proximity_to_simple_sensor or some other longer event name

Reply:
I disagree, but ok